src/CategoricalArrays.jl

```diff
@@ -21,6 +21,8 @@ module CategoricalArrays
     include("nullablearray.jl")
     include("deprecated.jl")
 
+    include("subarray.jl")
```

Comment:
Would be more logically placed before deprecated.jl (which could use a line break BTW).